lists.go

```diff
@@ -25,3 +30,19 @@ func (s *Session) GetUsers(l *List) (users []User, err error) {
 	err = s.do(rq).Into(&users)
 	return
 }
+
+// OptIn subscribes the given user to the given list
+// This consumes one API request.
+func (s *Session) OptIn(UserID int, ListID int) (err error) {
+	var body bytes.Buffer
+	var ok *bool
+	body.WriteString(fmt.Sprintf("userid=%d", UserID))
+	err = s.putf("/list/%d/optin", body, ListID).Into(&ok)
```

Comment:
Do `PUT`s always return an ok/not ok boolean?  If so, or at least if it's very common, there should probably be a nicer API than this for getting that response.  (Admittedly, that's my fault for making `get` do this :P)

Reply:
They do for [that endpoint](https://github.com/UniversityRadioYork/MyRadio/blob/master/src/Classes/ServiceAPI/MyRadio_List.php#L212). 
I did try several other ways first. They all didn't work. So if there is a better way then I've no clue how to do it.